source/unified-test-format/unified-test-format.md

```diff
@@ -2980,9 +3017,13 @@ Syntax:
 
 This operator may be used anywhere a matched value is expected (including [expectResult](#operation_expectResult)) and
 the actual value is an extended JSON string. The test runner MUST parse the actual value into a BSON document and match
-it against the expected value. Matching the expected value MUST use the standard rules in
-[Evaluating Matches](#evaluating-matches), which means that it may contain special operators. This operator is primarily
-used to assert on extended JSON commands and command replies included in log messages.
+it against the expected value.
+
+Matching the expected value MUST use the standard rules in [Evaluating Matches](#evaluating-matches), which means that
+it may contain special operators. The actual value MUST NOT be considered a root-level document (i.e. extra fields are
+not permitted); however, [$$matchAsRoot](#matchasroot) may be used to change that behavior.
```

Comment:
I dug up https://github.com/mongodb/specifications/pull/1303#discussion_r969045742 to get context on the relationship between `$$matchAsDocument` and `$$matchAsRoot`.

Also talked to @kmahar [in Slack](https://mongodb.slack.com/archives/C72LB5RPV/p1730132575920099) about this and I agree that there is better terminology than "root-level" to communicate whether extra fields are allowed or not, but we're probably stuck with it given the `$$matchAsRoot` operator. If there's ever consensus to change this down the line, I suppose we could introduce an alias and then revise the spec to replace all references to "root-level", but I'm not about to do that here.